.github/workflows/lint.yml

```
@@ -0,0 +1,93 @@
+name: Linting
+
+on: 
+  pull_request:
+    paths_ignore:
+      - "docs/**"
+
+defaults:
+  run:
+    # This is needed for miniconda, see:
+    # https://github.com/marketplace/actions/setup-miniconda#important
+    shell: bash -l {0}  
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    name: Linting checks
+    steps:
+      - uses: actions/checkout@v2.3.4
+        with:
+          submodules: recursive
```

To ensure the tags are retrieved fetch-depth needs to be set to 0, though I don't think it's all that relevant for this repo.
```suggestion
          submodules: recursive
          fetch-depth: 0
```

---

That will make it slower though. Especially for the Germline-DNA repo, and I intend to copy this file.